e2e/terraform/fedora/main.tf

```diff
@@ -0,0 +1,13 @@
+provider "google" {
+    credentials = file("/etc/satoken/satoken")
+    project = "pure-faculty-367518"
+    region = "us-central1"
+    zone = "us-central1-c"
```

Comment:
`project`, `region` and `zone` values can be placed in the variables file and have default values, in that way this could be simplified.

Reply:
When I depended on using the default values for variables defined in modules/e2e/variables.tf , `terraform apply` exited complaining that the the above mentioned values are required. Looks like we need to move the variables file out of the modules/e2e/ directory to a higher level directory. Would like to do this as part of a later PR